### PR TITLE
postinst : add enabled service in single REST API call and other textual fixes

### PR DESCRIPTION
**Pull request by praveen-garg**

### CODE REVIEW

File: south_plugin_packager/packages/Debian/armhf/DEBIAN/postinst [hunk 1]
```diff
@@ -32,19 +32,10 @@ set_files_ownership () {
...
-    output=$(curl -sX POST http://localhost:8081/foglamp/service -d '{"name": "poll_template", "type": "south", "plugin": "poll_template"}')
-    service_id=$(echo $output | awk -F '"id": ' '{print $2}' | awk -F \" '{print $2}')
-    echo $service_id
-}
-
-enable_service () {
-    service_id=$1
-    echo $service_id
-    output=$(curl -X PUT -d '{"enabled": true}' http://localhost:8081/foglamp/schedule/$service_id)
+    output=$(curl -sX POST http://localhost:8081/foglamp/service -d '{"name": "poll_template", "type": "south", "plugin": "poll_template", "enabled": true}')
```
Comment (ashish-jabble) [inline south_plugin_packager/packages/Debian/armhf/DEBIAN/postinst]: name: Poll Template

File: south_plugin_packager/packages/Debian/x86_64/DEBIAN/postinst [hunk 2]
```diff
@@ -32,19 +32,10 @@ set_files_ownership () {
...
-    output=$(curl -sX POST http://localhost:8081/foglamp/service -d '{"name": "poll_template", "type": "south", "plugin": "poll_template"}')
-    service_id=$(echo $output | awk -F '"id": ' '{print $2}' | awk -F \" '{print $2}')
-    echo $service_id
-}
-
-enable_service () {
-    service_id=$1
-    echo $service_id
-    output=$(curl -X PUT -d '{"enabled": true}' http://localhost:8081/foglamp/schedule/$service_id)
+    output=$(curl -sX POST http://localhost:8081/foglamp/service -d '{"name": "poll_template", "type": "south", "plugin": "poll_template", "enabled": true}')
```
Comment (ashish-jabble) [inline south_plugin_packager/packages/Debian/x86_64/DEBIAN/postinst]: name: Poll Template

**Review by ashish-jabble — CHANGES_REQUESTED** [review]
While making deb, I got an error : 
```
dpkg-deb: error: package name has characters that aren't lowercase alphanums or '-+.'
```
It may be the issue regarding package name we have `foglamp-south-poll_template-${version}-${architecture}"`

Also fix below items in make_deb file:
i) sed -i "s/Version: 1.0.0/Version: $version/g" DEBIAN/control
ii) cp -R ${GIT_ROOT}/../south_plugin_poll/python .
* Also please change plugin description in poll_template.py

**Comment by praveen-garg** [comment]
@ashish-jabble  Fixed. including the control file. 

Regrading copy the plugin files, added comment; as these are example repositories.

File: south_plugin_packager/make_deb [hunk 3]
```diff
@@ -108,11 +108,13 @@ mkdir "${package_name}"
...
 cp -R ${GIT_ROOT}/packages/Debian/${architecture}/* .
-sed -i "s/Version: 0.0/Version: $version/g" DEBIAN/control
+sed -i "s/Version: 1.0.0/Version: $version/g" DEBIAN/control
 sed -i "s/Depends: foglamp/Depends: foglamp (=$foglamp_version)/g" DEBIAN/control
 
 mkdir -p usr/local/foglamp
 cd usr/local/foglamp
+# Make sure to copy python directory with tree structure
+# `python/foglamp/plugins/south/poll_template/..` in the GIT_ROOT directory
 cp -R ${GIT_ROOT}/python .
```
Comment (ashish-jabble) [inline south_plugin_packager/make_deb:118]: make_deb will fail at this point.

**Review by ashish-jabble — APPROVED** [review]

**Pull request merged**